### DISCUSSION TITLE
Support style options

### DISCUSSION THREAD

**Discussion by dhoepelman** [scala-tsi/scala-tsi#194] [Ideas]
* [ ] 2 or 4 spaces in typescript file
* [ ] emit `member: Option[T]` as `member: (T | undefined)` or `member?: T`

**Comment by carueda** [comment]
Perhaps also:

- [ ] No prefix  for generated interfaces; or
- [ ] Alternate prefix for generated interfaces: `_____`

(that is, a way to avoid the `I` prefix currently generated)
